examples/http-proxy/README.md

```diff
@@ -46,7 +43,7 @@ libp2p-peer addresses:
 The run the local peer, indicating that it will need to forward http requests to the remote peer as follows:
 
 ```
-$ ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
+> ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
```

Comment:
There was a conversation a long time ago between Juan and Richard where they questioned what was the most valid character to represent the prompt and the result of the conversation was `>`.

As for `#`, we use it everywhere as comments since it is the character for line comments in bash.